lib/times.js

```diff
@@ -40,7 +40,9 @@ function create (types) {
 }
 
 var times = {
-  week: function ( ) { return create('weeks')(1); }
+	mins_msecs: function (value) { return value*60*1000; }
```

Comment:
Maybe we can come up with better names for `mins_msecs` and `msecs_mins`, so it will be easier to remember what they do and will prevent the from getting used backwards